packages/core/src/domain/configuration.ts

```diff
@@ -143,6 +152,13 @@ export function buildConfiguration(initConfiguration: InitConfiguration, buildEn
     configuration.actionNameAttribute = initConfiguration.actionNameAttribute
   }
 
+  if (
+    'initialPrivacyLevel' in initConfiguration &&
+    objectValues(InitialPrivacyLevel).indexOf(initConfiguration.initialPrivacyLevel) >= 0
+  ) {
+    configuration.initialPrivacyLevel = initConfiguration.initialPrivacyLevel!
```

Comment:
This code is super cryptic to me, I would refactor or at least add a comment.
From my understanding, since the user is not using typescript we need to check at runtime that the `initialPrivacyLevel` is in the config & it's a valid value. We cannot use [`includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) because of IE11 ?
Then, we add with a `!` I do not get it. It's supposed to be a string

Reply:
The `!` is there to tell Typescript that we are sure that `initConfiguration.initialPrivacyLevel` is not `undefined`. Otherwise it will complain about assigning `undefined` to a field of type `InitialPrivacyLevel`.

Reply:
That's right, we can't use `includes` because of IE11. `indexOf >= 0` is super idiomatic in JS, so I thought that was fine. I'll see what I can do to improve the situation.